kernel-kmp-script

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
I think we should integrate this code entirely into `weak-modules2`.

Reply:
So long as the interface used in here is sufficient the code can be moved between s-m-t scripts later.